.pre-commit-config.yaml

```diff
@@ -55,6 +55,29 @@ repos:
       rev: 7.3.0
       hooks:
       -   id: flake8
+  
+  -   repo: https://github.com/pycqa/flake8
+      rev: 7.3.0
+      hooks:
+      -   id: flake8
+          name: flake8-docstrings with ament-style rules
```

Comment:
If you ran ament_flake8 as a pre-commit rule, would that replace the need to manually specify all the ignores on line 69?